### ISSUE TITLE
unsorting phenotype file results in inflation of p-values??

### ISSUE THREAD

**Issue by Pamelanmrc** [DavisBrian/seqMeta#29]
Hi Brian,

I have been running seqMeta for a while and I noticed something. When my phenotype input file is sorted my singlesnp results do not have any inflation in the p-value qq-plots. But if my phenotype file is not sorted, it does give me some degree inflation in the qq-plots. It still gives me the same significant snps albeit lower p-values. I checked my genotype matrix ordering (Which should be similar to phenotype sample order) just to make sure that nothing was wrong on my part. Is there something in seqMeta that requires a sorted phenotype file?
You insight would be very helpful.

Thanks.
P

**Comment by DavisBrian** [comment]
@Pamelanmrc Yes, it's assumed that the phenotypes are in the same order as the genotypes.  In the current implementation seqMeta has no way of knowing the order or your phenotypes.

**Comment by DavisBrian** [comment]
@Pamelanmrc  I'm going to leave this issue open for now.  Looking through the documentation I don't think this point is abundantly clear.  In the next release I'll update the documentation to make this point clear.  The underlying issue is neither the phenotype nor the genotype data explicitly has a subject identifier in the data.  In practice (at least with CHARGE) it was common practice to use the row names as the subject identifier and do the matching in the analysis pipeline before using prepScores.

I'm not a fan of using row names as a variable place holder since they tend to get dropped by various functions with little rationale on how and when.  For the phenotypes it's easy to enforce a subject_id column, but for genotypes it's a bit more troublesome without changing the data format altogether.
